### PR TITLE
Settings screen

### LINKED ISSUE #47: ⚙️ Settings
Add a Settings view, with a ⚙️ icon to open it next to the logo on the Home view.

This Settings screen should contain:
- Links: FAQ/VaccinTracker/...
- An additional link to open the contributors view (See #122)
- Options to contact us and/or report a bug
- UI Settings: app theme (system/light/dark) and accessibility

### PR DESCRIPTION
**Pull request by pylapp**
## Description
Provide as described in issue #47 a setting screen for the application.
User can tap on a gear icon to go to settings screen.
This new view displays items about the official web site, contact form, Twitter profile, source code page an has also a link to make the user go to system app settings. Thus, once merge request #114 will be merged, users will be able to see all third party elements from there in one tap.
Issue #37 can also be integrated at this level with its own cell (see #122).

## Changes
- Gear icon from home screen
- Settings screen based on UITableView with cells
- Cell for web site
- Cell for contact form
- Cell for Twitter
- Cell for source code (GitHub)
- Cell for app settings
- VoiceOver vocalization (a11y)
- Dynamic fonts (a11y)

## Related issue

Fixes #47 

## What I tested

For a11y, used my own device with VoiceOver enabled and large text sizes.
Otherwise use the same device for navigation.
See screen shots below.

## Regression risks
- None

## Screenshots

<table>

<tr>
<td>

</td>
<td>

</td>
<td>

</td>
</tr>

<tr>
<td>

</td>
<td>

</td>
<td>

</td>
</tr>

<tr>
<td>

</td>
<td>

</td>
</tr>

</table>

## Checklist
- [x] I have installed SwiftLint and made sure code formatting is correct
- [x] I have performed a self-review of my own code
- [x] I tested my changes on real device
- [x] My changes generate no new warnings
- [x] I have commented my code in hard-to-understand areas
- [x] Any dependent changes have been merged into **develop**
- [x] I have checked there aren't other open [Pull Requests](https://github.com/CovidTrackerFr/vitemadose-ios/pulls) for the same update/change

### CODE REVIEW

**Comment by pylapp** [comment]
@victor-sarda This merge request for issue #47 is ready for review 👍

File: ViteMaDose.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1161,7 +1205,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 841539;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
-				DEVELOPMENT_TEAM = ZL9396WK65;
+				DEVELOPMENT_TEAM = "";
```
Comment (NathanFallet) [inline ViteMaDose.xcodeproj/project.pbxproj:1208]: Please remove that change from the PR (it should not be committed)

File: ViteMaDose.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1183,7 +1227,7 @@
 				CODE_SIGN_ENTITLEMENTS = ViteMaDose/ViteMaDose.entitlements;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 841539;
-				DEVELOPMENT_TEAM = ZL9396WK65;
+				DEVELOPMENT_TEAM = "";
```
Comment (NathanFallet) [inline ViteMaDose.xcodeproj/project.pbxproj:1230]: Same here

**Review by NathanFallet — CHANGES_REQUESTED** [review]
Seems good, except some unwanted changes in Xcode settings (see review comments) + maybe about analytics (saw your comments in code but don't know if needed or not, see this with @victor-sarda)

File: ViteMaDose.xcodeproj/project.pbxproj (hunk 2 above)
Comment (pylapp) [reply ViteMaDose.xcodeproj/project.pbxproj]: Now fixed 😄

**Comment by pylapp** [comment]
@NathanFallet About analytics I let @victor-sarda provide details, or add them later if needed 👍

**Review by NathanFallet — APPROVED** [review]

**Comment by pylapp** [comment]
@NathanFallet @victor-sarda I updated the feature branch, feel free to bring your reviews so that we can just after the merge integrate the work of Nathan 👍

**Comment by pylapp** [comment]
@NathanFallet @victor-sarda Any way to make this request be merged?
Do you need some help?

Cheers 😄

**Comment by victor-sarda** [comment]
> @NathanFallet @victor-sarda Any way to make this request be merged?
> Do you need some help?
> 
> Cheers 😄

Sorry, again I have almost no time to review/work on the app right now. I'm not 100% sold on this to be honest. Does it add a real value to the app? It's mainly external links, not really settings. Maybe we should discuss what we want to do for a settings screen. Some accessibility settings for instance? Or options like enable/disable dark mode? 
Feel free to join our Mattermost if you want to discuss about it :)

**Comment by pylapp** [comment]
Indeed I may join your Mattermot team.

In a nutshell, I don't think it is a good idea to override some system settings like light/dark mode or define accessibilty settings, the system does.
In the user story #47 there were only just ideas I tried to implement. Having specific items about social network, source code and contributors may be quite cool.
Maybe should we name it "about" and not "settings" 😃

**Review by NathanFallet — APPROVED** [review]

**Review by victor-sarda — APPROVED** [review]
Nice one!

**Pull request merged**